packaging/python/build_wheels.bash

```diff
@@ -109,7 +109,7 @@ build_wheel_osx() {
     if [ "$2" == "--bare" ]; then
         python setup.py bdist_wheel
     else
-        python setup.py build_ext --cmake-defs="NRN_MPI_DYNAMIC=$3" bdist_wheel
+        python setup.py build_ext --cmake-prefix="/usr/local/opt/ncurses;/usr/local/opt/readline" --cmake-defs="NRN_MPI_DYNAMIC=$3" bdist_wheel
```

Comment:
for the wheels, general rule is do not use stuff from system i.e. /usr. We should discuss this.

Reply:
That is homebrew's path

Reply:
discussed offline: To make sure 100% we have the proper MACOSX_DEPLOYMENT_TARGET, we should build `readline` and `ncurses` and have them available somewhere.